### DISCUSSION TITLE
Estep Calculator

### DISCUSSION THREAD

**Discussion by hpolack** [mriscoc/Ender3V2S1#1225] [Ideas]
I finally made the switch from Jyer for my V2 and let me just say WOW what fantastic tools for leveling.   The incredible amount of configuration and ease to access is fantastic.  The only thing I was missing on tools is the eStep calculator I have in my Community 6.1 firmware on my CR6 SE.   I of course can do the calculations, but the way they have the estep tool with calculator and instructions on what to do, make it easy even for a beginner who shy away from these things because they seem scary.  If Mriscoc added that, this would be the best I have seen in tools to set leveling, PID bed & hot end and hopefully soon to be eStep calculator.

**Comment by mriscoc** [comment]
That could be a good addition but only for RET6 devices which have the enough program memory space available.
